js/context-date.js

```diff
@@ -0,0 +1,36 @@
+// Can use Symbol, but let's not make this more complex.
+/**
+ * Context Request event for signaling we want to get other formats of the same date.
+ */
+export const ContextRequest_DateConversion = 'date-conversion'
+
+const KEYS_DateConversion = ['date', 'dateIsoString', 'dateUnix', 'dateHuman']
+
+export const isContextRequest_DateConveresion = (event) =>
+  event.context === ContextRequest_DateConversion
+
+export const assertContextRequest_DateConveresion = (event) => {
+  if (!isContextRequest_DateConveresion(event)) {
+    const message = `Unexpected error, we expected a "ContextRequest_DateConversion" context event`
+    throw new Error(message)
+  }
+}
+
+export const getPropertiesFrom_DateConversion = (event) => {
+  assertContextRequest_DateConveresion(event)
+  const maybeDate = event.originalTarget.getAttribute('date')
+  const date = maybeDate ?? null
+  const dateHumanFormat = 'MMM D, YYYY' // TODO Make configurable from event.target
+  return Object.freeze({
+    date,
+    dateHumanFormat,
```

Comment:
To allow specifying own format.   Maybe we want more than one formats configurable.